presentations/index.md

```diff
@@ -68,41 +71,51 @@ Open Source Backup Conference @ Cologne, Germany
+    
+2015 Storage Developer Conference 
+--------------------------------------
+
+-  Achieving Coherent and Aggressive Client Caching in Gluster, a Distributed System
+[pdf](http://www.snia.org/sites/default/files/SDC15_presentations/file_sys/Poornima_Sourmya_Achiving_Coherent_Aggressive_Client-2.pdf) - Poornima Gurusiddaiah, Soumya Koduri
+
+- Introduction to Highly Available NFS Server on Scale-Out Storage Systems Based on GlusterFS
+[slideshare](http://www.slideshare.net/GlusterCommunity/introduction-to-highlyavailablenfsserveronscaleoutstoragesystemsbasedonglusterfssdc2015) - Soumya Koduri, Meghana Madhusudhan
+
 
 Gluster Summit 2015 @ Barcelona, Spain
 --------------------------------------
 
--   [Bug Triage in Gluster](./Bug_Triage_in_Gluster.odp) -
+-   [Bug Triage in Gluster](http://www.slideshare.net/GlusterCommunity/bug-triage-ingluster) -
     Niels de Vos
 -   [Responsibilities of Gluster
-    Maintainers](./Responsibilities_of_Gluster_Maintainers.odp) -
+    Maintainers](http://www.slideshare.net/GlusterCommunity/responsibilities-of-glustermaintainers) -
     Niels de Vos
--   [Leases and caching](./Leases-and-caching_final.odp) -
+-   [Leases and caching](http://www.slideshare.net/GlusterCommunity/leases-andcaching-final) -
     Poornima Gurusiddaiah & Soumya Koduri
 -   [Cache tiering in GlusterFS and future
-    directions](./Tiering_barcelona.odp) - Dan Lambright
+    directions](http://www.slideshare.net/GlusterCommunity/tiering-barcelona) - Dan Lambright
 -   [Yet Another Deduplication
-    Library (YADL)](./Light_barcelona.odp) - Dan Lambright
+    Library (YADL)](http://www.slideshare.net/GlusterCommunity/ydal-barcelona) - Dan Lambright
 
 Gluster Conference @ NMAMIT, Nitte
 ----------------------------------
 
 ### April 11, 2015
 
 -   [Introduction to Open
-    Source](./GlusterFS_Open_Source.odp) - Niels de Vos,
-    Red Hat
+    Source](http://www.slideshare.net/GlusterCommunity/introduction-to-open-source-62086626) - Niels de Vos, Red Hat
+    
 -   [Software Defined
-    Storage](./Software_Defined_Storage.odp) - Dan
+    Storage](http://www.slideshare.net/GlusterCommunity/software-defined-storage-62086656) - Dan
     Lambright, Red Hat
+    
 -   [Introduction to
-    GlusterFS](./Gluster_Technical_Overview.odp) -
-    Kaleb S. KEITHLEY, Red Hat
--   [Data Deduplication](./Dedupe_nmamit.odp) - Joseph
+    GlusterFS](http://www.slideshare.net/GlusterCommunity/gluster-technical-overview) -  Kaleb S. KEITHLEY, Red Hat
+    
+-   [Data Deduplication](http://www.slideshare.net/GlusterCommunity/dedupe-nmamit) - Joseph
     Fernandes, Red Hat
--   [Quality of Service](./Qos.odp) - Karthik US & Sukumar
+-   [Quality of Service](http://www.slideshare.net/GlusterCommunity/qos-62086668) - Karthik US & Sukumar
     Poojary, 4th SEM, MCA, NMAMIT, Nitte
 
 [Ceph & Gluster FS - Software Defined Storage MeetupCeph & Gluster FS - Software Defined Storage Meetup](http://www.meetup.com/RedHatFinland/events/218774694/)
```

Comment:
Unrelated to your PR. Duplicate content here. Making a comment here so that I don't forget.